src/UI/Component/Listing/SimpleList.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Listing;
+
+/**
+ */
+interface SimpleList extends \ILIAS\UI\Component\Component {
+	const UNORDERED = "ul";
+	const ORDERED = "ol";
+
+	/**
+	 * @param string[] $items
+	 * @return \ILIAS\UI\Component\Listing\SimpleList
+	 */
+	public function withItems(array $items);
+
+	/**
+	 * @return string[] $items
+	 */
+	public function getItems();
+
+	/**
+	 * @param int $type
+	 * @return \ILIAS\UI\Component\Listing\SimpleList
+	 */
+	public function withType($type);
+
+	/**
+	 * @return int
+	 */
+	public function getType();
```

Comment:
As per [Interfaces to UI components, No.2](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/src/UI/docu/rules.md#interfaces-to-ui-components) please either make all three types of list just be distinguished only by a type or all three lists separate classes, i.e. make an `Ordered` and `Unordered` interface. I would go with the second.

The reasoning behind that rule for me was, that it is harder to find out what is going on, when both Styles (getType/separate classes) are mixed in one sub namespace. This rule is of course debatable.